src/main/java/sirius/tagliatelle/macros/ShortenMacro.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.tagliatelle.macros;
+
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Register;
+import sirius.tagliatelle.Tagliatelle;
+import sirius.tagliatelle.expression.Expression;
+import sirius.tagliatelle.rendering.LocalRenderContext;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * Represents <tt>shorten(String, int)</tt> which is a call to {@link Strings#shorten(String, int)}.
+ */
+@Register
+public class ShortenMacro implements Macro {
```

Comment:
Functionality is already possible with the limit Macro

Reply:
but LimitMacro doesn't add ellipsis (3rd parameter is implicit false)